diagnostic.md

```diff
@@ -3,36 +3,44 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+Low bandwith environments with single page applications.
+You can control state with API only development.
+Can do away with jquery.
 ```
 
 What is the templating system used in Ember? How it is related to
 handlebars?
 
 ```sh
-# your answer here
+The templating system is handlebars. You load templates and components based on URLs.
```

Comment:
Yes. It also has some built in helpers, like `link-to` that aren't available in regular handlebars.
1/1